### PR TITLE
Multichain support

### PR DESCRIPTION
**Pull request by coozebra**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "@types/node": "~14.14.22",
     "@typescript-eslint/eslint-plugin": "~4.14.1",
     "@typescript-eslint/parser": "~4.14.1",
-    "@umb-network/toolbox": "~4.1.1",
+    "@umb-network/toolbox": "5.0.0-alpha.1",
```
Comment (DZariusz) [inline package.json:37]:
```suggestion
    "@umb-network/toolbox": "5.0.0",
```

I released non alfa, so we can update.

**Review by DZariusz — COMMENTED** [review]
lets add sandbox as well

**Review by DZariusz — APPROVED** [review]
looks good, please sync with Rodrigo to make sure it will be set for sandbox. But not a priority - if Rodrigo do not have time today, lets just create a task in Jira so we do not forget.

**Comment by DZariusz** [comment]
after merge, please make a release.

**Comment by coozebra** [comment]
Okay, I will do

**Pull request merged**